pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -229,14 +231,13 @@ func TestSingleZone(t *testing.T) {
 				nil,
 			}
 
-			_, gotPreFilterStatus := p.PreFilter(ctx, state, test.Pod)
-			if !gotPreFilterStatus.IsSuccess() {
-				if diff := cmp.Diff(gotPreFilterStatus, test.wantStatus); diff != "" {
-					t.Errorf("status does not match (-want,+got):\n%s", diff)
-				}
-			} else {
-				gotStatus := p.Filter(ctx, state, test.Pod, node)
-				if diff := cmp.Diff(gotStatus, test.wantStatus); diff != "" {
+			_, preFilterStatus := p.PreFilter(ctx, state, test.Pod)
+			if diff := cmp.Diff(preFilterStatus, test.wantPreFilterStatus); diff != "" {
+				t.Errorf("status does not match (-want,+got):\n%s", diff)
+			}
+			if preFilterStatus.IsSuccess() {
```

Comment:
I'd suggest always running Filter regardless of `preFilterStatus` so that we can confirm if it works as expected when only Filter is enabled.
Ditto for other tests in this file.